lib/common.js

```diff
@@ -160,11 +160,10 @@ function validateIsArray(array) {
 function validateNftId(nftId) {
   try {
     if (isHex(nftId)) return new BN(nftId.substring(2), 16).toString(10);
-    else {
-      const hexNftId = new BN(nftId).toString(16);
-      if (hexNftId.length <= 64) return nftId;
-      else throw nftId;
-    }
+    if (!nftId) throw nftId;
+    const hexNftId = new BN(nftId).toString(16);
```

Comment:
I would rewrite this as something like:
```
if (!nftId) throw nftId;

const hexNftId = isHex(nftId) ? nftId : new BN(nftId).toString(16);
if (hexNftId.length > 64) throw nftId;

return isHex(nftId) ? new BN(nftId.substring(2), 16).toString(10) : nftId

```

Reply:
Yep, we agreed the NFT ID would always be decimal (much to my chagrin)

Reply:
I'm with you on this, @fluorostani :grin: